pages/timeline/index.tsx

```diff
@@ -2,11 +2,25 @@ import type { NextPage } from "next";
 import "react-vertical-timeline-component/style.min.css";
 import React from "react";
 import Timeline from "../../components/Timeline";
-import timelineEvents from "../../data/timeline.json";
 import NavBar from "../../components/NavBar";
 import Header from "../../components/Header";
+import useSWR from "swr";
+import { TimelineEvent } from "models/schema";
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
 
 const TimelinePage: NextPage = () => {
+  const { data, error } = useSWR<TimelineEvent[], unknown>(
```

Comment:
Is there a reason you've put type=`unknown` here, or could you find out the correct type for `error` and use this instead?